torch/_inductor/cudagraph_trees.py

```diff
@@ -1022,15 +1060,16 @@ def check_memory_pool(pool_id, live_storages_ptrs: List[StorageWeakRefWrapper]):
     unique_storages = {stor.data_ptr() for stor in live_storages_ptrs if stor()}
     segments = get_cudagraph_segments(pool_id)
 
+    allocated_not_in_live_storages = []
+
     for segment in segments:
         addr = segment["address"]
         for block in segment["blocks"]:
             if block["state"] == "active_allocated":
-                check(
-                    addr in unique_storages,
-                    lambda: f"{addr} allocated but not in live storages",
-                )
-                unique_storages.remove(addr)
+                if addr not in unique_storages:
+                    allocated_not_in_live_storages.append(addr)
```

Comment:
I guess addresses isn't going to be terribly useful but I guess we don't have stack traces (or maybe we do, if @zdevito's thing is on?)

Reply:
Yea - I have a WIP branch that will add memory recording to improve error messages. Am working on it.